src/main/resources/avro/metadata.avdl

```diff
@@ -147,6 +148,8 @@ record Sample {
   /**
   The time at which this sample was taken from the individual, in milliseconds
   from the epoch.
+  TODO: should we consider using relative time (e.g. to the start of a study) rather than absolute time?
```

Comment:
Relative timestamps would be a source of error.  Additionally,
GMT should be required.  Timezone should only be a display issue.

Additionally, Avro 1.8 will include a data/time type:
              https://issues.apache.org/jira/browse/AVRO-739

Angel Pizarro <notifications@github.com> writes:
> In src/main/resources/avro/metadata.avdl:
> 
> > @@ -147,6 +148,8 @@ record Sample {
> >    /**
> >    The time at which this sample was taken from the individual, in milliseconds
> >    from the epoch.
> > +  TODO: should we consider using relative time (e.g. to the start of a study) rather than absolute time?
> 
> -1
> 
> I am against relative timestamps. This would complicate the gathering of
> metadata, and force a relationship between fields that don't really need to
> know about each other. In this case in particular, you would need to know the
> date of a whole other record, Experiment, to be able to interpret the date
> correctly.
> 
> —
> Reply to this email directly or view it on GitHub.*
>